### PR TITLE
crud

### PR DESCRIPTION
**Pull request by mrkent**

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -12,17 +13,66 @@ app.use(bodyParser.json());
...
-  const { id } = req.params;
-  PriceDataDay.find({ id }, null, { sort: { date: 1 } }, (error, results) => {
-    if (error) {
-      res.status(500).send(error);
-    } else {
-      res.status(200).send(results);
-    }
+    const resSender = (error, results) => {
+      if (error) {
+        res.status(500).send(error);
```
Comment (yukiej) [inline server/index.js]: Instead of res.send(error), would recommend using express error handling with "next"

File: server/index.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ const path = require('path');
 
 const { PriceDataDay } = require('../database/PriceDataDay.js');
 const db = require('../database/index.js');
+const con = require('../database/controllers.js');
```
Comment (yukiej) [inline server/index.js]: this is nice and concise, but might give this a slightly more descriptive variable name

File: server/index.js [hunk 3]
```diff
@@ -12,17 +13,66 @@ app.use(bodyParser.json());
...
 
-app.get('/api/symbol/:id/day', (req, res) => {
-  const { id } = req.params;
-  PriceDataDay.find({ id }, null, { sort: { date: 1 } }, (error, results) => {
-    if (error) {
-      res.status(500).send(error);
-    } else {
-      res.status(200).send(results);
-    }
+    const resSender = (error, results) => {
```
Comment (yukiej) [inline server/index.js]: Might pull this outside of the app.route, since it is currently identical for all routes

File: server/index.js [hunk 4]
```diff
@@ -12,17 +13,66 @@ app.use(bodyParser.json());
...
+        res.status(200).send(results);
+      }
+    };
+
+    con.create(item, resSender);
+  })
+  .put((req, res) => {
+    const { id } = req.params;
+
+    const item = Object.assign({}, req.body);
```
Comment (yukiej) [inline server/index.js]: es6 pro tip: You can shorten this to: 
const item = { ...req.body }

File: server/index.js [hunk 5]
```diff
@@ -12,17 +13,66 @@ app.use(bodyParser.json());
...
+        res.status(200).send(results);
+      }
+    };
+
+    con.remove(id, resSender);
   });
-});
+
 
 app.get('/api/symbol/:id/week', (req, res) => {
```
Comment (yukiej) [inline server/index.js]: If you are refactoring the routes above with app.route, would recommend doing it for the rest of the routes too to have less work when you swap out your database

**Review by yukiej — COMMENTED** [review]
Looks good overall. Have a few minor suggestions, let me know if you have any questions!

File: server/index.js (hunk 3 above)
Comment (mrkent) [reply server/index.js]: I wanna change this at some point.

File: server/index.js (hunk 5 above)
Comment (mrkent) [reply server/index.js]: this will not be used for now. will comment out
